docs/design/specs/PortablePdb-Metadata.md

```diff
@@ -598,3 +599,22 @@ The order of the options in the list is insignificant.
 > The purpose of this data is to allow a tool to reconstruct the compilation the module was built from.
 > The source files for the compilation are expected to be recovered from the source server using [SourceLink](#source-link-c-and-vb-compilers) and/or from [sources embedded](#embedded-source-c-and-vb-compilers) in the PDB.
 > The metadata references for the compilation are expected to be recovered from a file indexing service (e.g. symbol server) using information in [Compilation Metadata References](#compilation-metadata-references-c-and-vb-compilers) record.
+
+##### Type Definition Document (C# and VB compilers)
+Parent: TypeDef
+
+Kind: {932E74BC-DBA9-4478-8D46-0F32A7BAB3D3}
+
+Stores document information for type definitions that would otherwise not have document information stored through other means, for example in the [MethodDebugInformation](#methoddebuginformation-table-0x31) table.
+
+The blob has the following structure:
+
+    Blob ::= (document-ordinal)*
+
+| terminal         | encoding                    | description                            |
+|:-----------------|:----------------------------|:---------------------------------------|
+| document-ordinal | compressed unsigned integer | 1-based index into the Document table. |
```

Comment:
I'd use the term "row id" instead of "1-based index" since that's how ECMA calls these indices.